apps/web/src/state/predictions/helpers.ts

```diff
@@ -86,11 +88,32 @@ export const getRoundResult = (bet: Bet, currentEpoch: number): Result => {
     return Result.LIVE
   }
 
+  // House Win would not occur in AI-based predictions, only in normal prediction feature
   if (bet?.round?.position === BetPosition.HOUSE) {
     return Result.HOUSE
   }
 
-  const roundResultPosition = Number(round?.closePrice) > Number(round?.lockPrice) ? BetPosition.BULL : BetPosition.BEAR
+  let roundResultPosition: BetPosition
+
+  // If AI-based prediction.
+  if (round?.AIPrice) {
+    if (
+      // Result: UP, AI Voted: UP => AI Win
+      (Number(round?.closePrice) > Number(round?.lockPrice) && Number(round.AIPrice) > Number(round.lockPrice)) ||
```

Comment:
can get @chefjackson  as second opionion on this, but thought id just raise anyway. 

Sometimes its common when doing math aiirthmetic operations like `mul, add, div, sub gte etc...` throught the interface to use BigNumber notation sometimes. could we use notation 
```ts
new BigNumber(round.closePrice).gte(new BigNumber(round.lockPrice)) && .....
```
instead for here and other places math airthmetic is required?

Reply:
Here the round prices are already numbers. 
Do we want to convert numbers to BigNumber for math operations? Won't make a difference in precision, can be good for convention.
If so, I'll change it in other places too. (plenty of operations there in prediction 😄)

Reply:
i just said id raise this initially just from more of a conventional viewpoint. i think best to get jackson for best advice on this. if the cacls are not needed for really exact precision such as the likes of farm calulations etc, then i think it should fine to keep as is then

cc @chefjackson